File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -799,6 +799,116 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Replace.docx", "applic
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Access the specific paragraph in the Word document.
+        TextBodyItem textBodyItem = document.Sections[0].Paragraphs[3] as WParagraph;
+        //Find the next occurrence of specified text from the previous paragraph.
+        TextSelection textSelections = document.FindNext(textBodyItem, new Regex("Adventure Works Cycles"));
+        //Replace the text extended to two paragraphs with the above selected text.
+        document.ReplaceSingleLine("CompanyName", textSelections, true, true);
+        //Save the Word document to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        outputStream.Position = 0;
+        //Download as Word document in the browser.
+        return File(outputStream, "application/msword", "Sample.docx");
+    }
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+    //Load the file stream into a Word document.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Access the specific paragraph in the Word document.
+        TextBodyItem textBodyItem = document.Sections[0].Paragraphs[3] as WParagraph;
+        //Find the next occurrence of specified text from the previous paragraph.
+        TextSelection textSelections = document.FindNext(textBodyItem, new Regex("Adventure Works Cycles"));
+        //Replace the text extended to two paragraphs with the above selected text.
+        document.ReplaceSingleLine("CompanyName", textSelections, true, true);
+        //Save the Word document to the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing. 
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+	//Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+	//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+    }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Find-next-and-replace-with-formatted-text).
+
 ## Find and replace text with other text
 
 You can find text in a Word document and replace it with other text. Unlike the `Find` method, the Replace method replaces all occurrences of the text. You can customize it to replace only the first occurrence of a text by setting the `ReplaceFirst` property of the WordDocument class to true.
```

Comment:
Find a text in a Word document and replace it with other text. Unlike the `Find` method, the Replace method replaces all occurrences of the text. Customize it to replace only the first occurrence of a text by setting the `ReplaceFirst` property of the WordDocument class to true.

Reply:
@Chrispineagunja -This is not part of my task so you should not review it